src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.cpp

```diff
@@ -732,8 +767,21 @@ bool QMCFixedSampleLinearOptimizeBatched::processOptXML(xmlNodePtr opt_xml,
   vmcEngine.reset(nullptr);
 
   // Explicitly copy the driver input objects since they will be used to instantiate the VMCEngine repeatedly.
+ //Overwriting input information is also done here to account for the hybrid method 
   QMCDriverInput qmcdriver_input_copy = qmcdriver_input_;
   VMCDriverInput vmcdriver_input_copy = vmcdriver_input_;
+  
+  if(MinMethod == "hybrid")
+  {
+  qmcdriver_input_copy.readXML(hybridEngineObj->getSelectedXML());
```

Comment:
This is some sort of hack, not in the spirit of the input code, and processOptXML is just a put method but even less clear.